src/components/ResultsStats/ResultsStats.tsx

```diff
@@ -101,6 +102,10 @@ export function WaferStats(props: {
 						"%",
 					)}
 				</li>
+				<li className="result-stats__result result-stats__result--die-cost">
+					<DollarIcon />
+					Die Cost: {displayValue(props.results?.dieCost)}
```

Comment:
Couple of minor copy suggestions to improve clarity here:

- Use 'Cost Per Die' since die can be singular and plural
- Prepend '$' to the value
```suggestion
					Cost Per Die: {`$${displayValue(props.results?.dieCost)}`}
```